src/DIRAC/Core/Security/VOMS.py

```diff
@@ -74,6 +75,7 @@ def getVOMSAttributes(self, proxy, switch="all"):
                     if value and value not in attributes and value in validVOMSAttrs:
                         attributes.append(value)
 
+        returnValue = None
```

Comment:
Why not add `raise NotImplementedError(switch)`?

Reply:
I am happy with that too. This argument is actually never used :shrug: